keycloak.tf

```diff
@@ -81,6 +82,7 @@ resource "aws_ecs_task_definition" "keycloak-ecs-taskdef" {
       ],
       "secrets": [
         {"name":"KEYCLOAK_ADMIN_PASSWORD", "valueFrom":"${aws_secretsmanager_secret.keycloak-admin-password.arn}"},
+        {"name":"KC_BOOTSTRAP_ADMIN_PASSWORD", "valueFrom":"${aws_secretsmanager_secret.keycloak-admin-password.arn}"},
```

Comment:
Question: Should this be `default_admin_password`? I'm a little unsure how the bootstrap password will differentiate from the long-term password defined above